dataeng/jobs/analytics/MonitorBigqueryEventLoading.groovy

```diff
@@ -0,0 +1,34 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+
+class MonitorBigqueryEventLoading {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("monitor-bigquery-loading") {
+            logRotator common_log_rotator(allVars)
+            parameters {
+                stringParam('TOOLS_REPO', allVars.get('TOOLS_REPO_URL'), '')
+                stringParam('TOOLS_BRANCH', allVars.get('TOOLS_BRANCH'), 'e.g. tagname or origin/branchname')
+                stringParam('NOTIFY', '$PAGER_NOTIFY', 'Space separated list of emails to send notifications to.')
```

Comment:
Is this value used? I see that `common_publishers` is no longer imported at the top of the file.

Reply:
Definitely seems unused, I'll remove.  IIRC, the plan was to alarm via cloudwatch threshold.